charts/rstudio-pm/README.md.gotmpl

```diff
@@ -19,6 +19,26 @@ This chart requires the following in order to function:
     mount them into the container by specifying the `pod.volumes` and `pod.volumeMounts` parameters.
   * If you cannot use a `PersistentVolume` to properly mount your data directory, you'll need to mount your data in the container
     by using a regular [Kubernetes Volume](https://kubernetes.io/docs/concepts/storage/volumes), specified in `pod.volumes` and `pod.volumeMounts`.
+  * Alternatively, S3 storage can be used. See the next section for details.
+
+## S3 Configuration
+
+Package Manager can be configured to store data in S3 buckets (see the [Adming Guide](https://docs.rstudio.com/rspm/admin/files-directories/#data-destinations)
+for more details). When configured this way, AWS access credentials are required. You must set the `awsAccessKeyId` and `awsSeretAccessKey` chart values
```

Comment:
The property for the secret is misspelled, although I don't know if it's only misspelled here or globally.

Reply:
Thanks Jon. Fixed it up. My keyboard seems to drop keystrokes fairly frequently 😕  I need to get spellcheck working (can be a pain on Linux, but it's time).